core/workspace.js

```diff
@@ -133,6 +140,14 @@ Blockly.Workspace.prototype.isClearing = false;
  */
 Blockly.Workspace.prototype.MAX_UNDO = 1024;
 
+/**
+ * Adds cursor for keyboard navigation.
+ * @return {Blockly.Cursor} Cursor for keyboard navigation.
+ */
+Blockly.Workspace.prototype.addCursor = function() {
```

Comment:
nit: s/add/new. This method is just creating the cursor and returning it, it doesn't add it to anything.

Should also be protected.